src/docs/patterns/forms.mdx

```diff
@@ -74,14 +103,97 @@ Some fields should verify the user input as it changes. The most common case for
 
 <ImageGrid
     images={[OnPageChangeImage]}
-    gridImageProps={{ sm: 12 }}
+    regularWidth
     caption={[
         'After the user clicks the submit button, the app waits for the server response, and displays an error message.',
         '1: The helper text will be replaced by the error message. ',
         '2: Use a circular progress indicator when waiting for responses from the server.',
     ]}
 />
+
+## Submitting the Form
+
+Typically, forms should have a button to submit the data or continue to the next part of the form (for long forms). Depending on what happens next, the button should be labeled "next", "finish", or similar. The button should be located at the bottom right corner of the form (bottom left for right-to-left languages).
+
+<ImageGrid
+    images={[SumbitButtonLocationImage]}
+    regularWidth
+    caption={[
+        '1: On desktops, the button is placed in the bottom corner of the form.',
+        '2: On mobile devices, the button is replaced by a button on native keyboard.',
+    ]}
+/>
+
+On mobile, if there are multiple input fields on the screen, the keyboard button should say "next", unless you are on the last item of the page, in which case the keyboard button should say "done" or "submit". In this situation, depends on if you want your user to double check their input, either dismiss the keyboard upon clicking the button (blur away from the input field), or transition the UI to the next page (submit the form directly).
+
+<ImageGrid
+    images={[NextButtonUsageImage]}
+    regularWidth
+    caption={[
+        ,
+        '1: The user entered a valid input. Clicking "next" will autofocus the user to the next text field.',
+        '2: The user entered an invalid input. Clicking "next" will display the error message but will not autofocus to the next text field.',
+    ]}
+/>
+
+For screens like email verifications where there is only one input field with a specific length, and the user will almost never need to go back and double check their input, the submit button can be omitted.
+
+<ImageGrid
+    images={[SubmitButtonAutoSubmitImage]}
+    regularWidth
+    caption={
+        'For the verification code screen, you can auto-submit the information once the user input reaches the required length (#2). After the verification code is recognized (#3), the screen pauses for a short moment before automatically moving to the next screen.'
+    }
+/>
+
+### Disabling the Submit Button
+
+In most cases, the submit button **should not** be disabled. The users should not be forced to search for what they are missing, or what they did wrong.
```

Comment:
Don't they still have to search if the submit button is enabled? Won't the error messages be there even when the submit button is disabled?

Reply:
Applies to people who skipped a field.